src/test/java/io/snyk/jenkins/SnykStepBuilderDescriptorTest.java

```diff
@@ -66,11 +66,11 @@ public void doCheckSeverity_shouldReturnWarning_ifProjectNameOverriddenWithAddit
   }
 
   @Test
```

Comment:
Could we also test that envvar is read?

Reply:
Just repeating myself here from the previous comment for completeness:

> No, as build environment variables are only known at build time (they're dynamic). `doCheckSnykTokenId` is only enforced at the configuration form interface.